### PR TITLE
feat(Dev/creatorName)

### PR DESCRIPTION
**Pull request by satyamm009**
### Functionality:
Creator field is initialized with the user(creator) email.

### Solution:
Fetched the user email from Auth Services and assigned it to creatorName.

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
firebase emulators:start
ng serve --open

creatorName2

### CODE REVIEW

**Comment by k0malSharma** [comment]
Resolve merge conflict

File: src/app/body/create-new-session/create-new-session.component.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ import { ApplicationSettingsService } from 'src/app/services/application-setting
 import { AuthService } from 'src/app/services/auth.service';
 import { Tasks } from 'src/app/Interface/TasksInterface';
 import { CloneTaskService } from 'src/app/services/clone-task.service';
+import { user } from 'firebase-functions/lib/providers/auth';
```
Comment (k0malSharma) [inline src/app/body/create-new-session/create-new-session.component.ts:16]: Use the available auth services.

File: src/app/body/create-new-session/create-new-session.component.ts [hunk 2]
```diff
@@ -29,7 +30,7 @@ export class CreateNewSessionComponent implements OnInit {
   todayDate: string
   description: string
   assigneeName: string
-  creatorName: string
+  creatorName = this.authService.user.email
```
Comment (k0malSharma) [inline src/app/body/create-new-session/create-new-session.component.ts:33]: Add the specification and assign an empty element. And use authService in ngOnInit()

**Review by k0malSharma — CHANGES_REQUESTED** [review]
Resolve merge conflict and make the requested changes

**Comment by satyamm009** [comment]
https://user-images.githubusercontent.com/74500169/129450146-43c4d179-f3e8-4306-926c-877af02d49a3.mp4


Are there any changes needed?

**Comment by viv9k** [comment]
It's not about just completing but doing it the right way. Please fix the changes requested by komal

File: src/app/body/create-new-session/create-new-session.component.ts (hunk 2 above)
Comment (satyamm009) [reply src/app/body/create-new-session/create-new-session.component.ts:33]: creatorName

Now, are these changes enough?

File: src/app/body/create-new-session/create-new-session.component.ts [hunk 3]
```diff
@@ -12,7 +12,8 @@ import { BackendService } from 'src/app/services/backend/backend.service';
 import { ApplicationSettingsService } from 'src/app/services/applicationSettings/application-settings.service';
 import { AuthService } from 'src/app/services/auth.service';
 import { Tasks } from 'src/app/Interface/TasksInterface';
-import { CloneTaskService } from 'src/app/services/cloneTask/clone-task.service';
+import { CloneTaskService } from 'src/app/services/clone-task.service';
```
Comment (k0malSharma) [inline src/app/body/create-new-session/create-new-session.component.ts:15]: Incorrect location. Correct the path.

**Review by k0malSharma — CHANGES_REQUESTED** [review]

File: src/app/body/create-new-session/create-new-session.component.ts [hunk 4]
```diff
@@ -29,7 +29,7 @@ export class CreateNewSessionComponent implements OnInit {
   todayDate: string
   description: string
   assigneeName: string
-  creatorName: string
+  creatorName = this.authService.user.email
```
Comment (k0malSharma) [inline src/app/body/create-new-session/create-new-session.component.ts:32]: Add the "string" specification. On ngOnInit you are calling the getUserEmail() function in auth service. So as explained before, do not assign value before the constructor

**Review by k0malSharma — CHANGES_REQUESTED** [review]

**Review by k0malSharma — APPROVED** [review]
Ship it

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**